subsys/mgmt/mcumgr/smp_shell.c

```diff
@@ -138,8 +142,15 @@ static uint16_t smp_shell_get_mtu(const struct net_buf *nb)
 
 static int smp_shell_tx_raw(const void *data, int len, void *arg)
 {
-	/* Cast away const. */
-	k_str_out((void *)data, len);
+	const struct shell_uart *const su = shell_backend_uart_get_ptr();
```

Comment:
I wish this idiom of adding const to local variables wasn't being used.  It's not common in Zephyr, and creates difficulties when using Coccinelle to do massive changes involving `const` like #24873 

However there's enough of this already in-tree it's not worth changing IMO.  We might have to deal with it globally some day.

Reply:
I do not know, I kinda like it. It just tells me that I am just using this constant as an shortcut/alias only.
It helped me to find bug once when I have used `=` instead of `==`  in conditional expression, in the old times.